src/pydvl/utils/parallel/backend.py

```diff
@@ -162,12 +158,11 @@ class RayParallelBackend(BaseParallelBackend, backend_name="ray"):
     """
 
     def __init__(self, config: ParallelConfig):
-        config_dict = asdict(config)
-        config_dict.pop("backend")
-        n_cpus_local = config_dict.pop("n_cpus_local")
-        if config_dict.get("address", None) is None:
-            config_dict["num_cpus"] = n_cpus_local
-        self.config = config_dict
+        self.config = {"logging_level": config.logging_level}
```

Comment:
The issue was created due to the modification of the temp dir. In that case where would we add it, as a separate parameter to __init__ or to ParallelConfig?

Reply:
Is this comment for the [class-wise shapley PR](https://github.com/appliedAI-Initiative/pyDVL/pull/338)?